pipelines/skylab/smartseq2_single_nucleus_multisample/Loom_schema.md

```diff
@@ -0,0 +1,117 @@
+# What's in the Smart-seq2 Multi Sample Pipeline Loom File?
```

Comment:
Is this already updated to reflect the outputs for single nuclei?

Reply:
This file is legacy and needs to be deleted. Loom schemas are moved to our doc site.